src/components/progressbar.tsx

```diff
@@ -17,21 +17,46 @@
  */
 
 import "../css/progressbar.css";
-import React from "react";
+import React, { useContext } from "react";
+import { ConfigContext } from "../config";
+import { Status } from "../rpc/transmission";
 
 interface ProgressBarProps {
     now: number,
     max?: number,
     label?: string,
     animate?: boolean,
+    status?: number,
     className?: string,
 }
 
 export function ProgressBar(props: ProgressBarProps) {
     const max = props.max ?? 100;
     const percent = Math.floor(1000 * props.now / max) / 10;
-    const label = props.label ?? `${percent}%`;
-    const className = `progressbar ${props.animate === true ? "animate" : ""} ${props.className ?? ""}`;
+    const label = props.label || `${percent}%`;
+    const animate = (props.animate && props.status !== Status.queuedToVerify &&
```

Comment:
Animate prop should also be taken as is. Determine whether progress bar should be animated in the parent.